Bottom: b9999f8d2f0740d11de00c17bb5b7ae8019f257e
Top:    3a13fcf6519ed5b87e09fca675288b5fb1d20a14
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 05:40:27 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
